src/service/tags/tags_repository.py

```diff
@@ -0,0 +1,99 @@
+from app_names import TableNames
+from query_builder import QuerySQLBuilder
+from response_sql import ResponseSQL
+
+
+class TagsRepository:
+    def __init__(
+        self, session, query_builder: QuerySQLBuilder, response_sql: ResponseSQL, logger
+    ) -> None:
+        self.logger = logger
+        self.session = session
+        self.query_builder = query_builder
+        self.response_sql = response_sql
+
+    def get_total_number_of_tags(self) -> dict:
+        try:
+            query = (
+                self.query_builder.add_table_name(TableNames.TAG)
+                .add_select_conditions(["COUNT(*)"])
+                .build()
+                .get_query()
+            )
+            result = self.session.execute(query).fetchall()
+            return self.response_sql.process_query_result(result)
+        except Exception as error:
+            self.logger.error(error)
+            raise error
+
+    def __get_subquery_tag(self, offset: int, max_count: int) -> str:
+        return (
+            self.query_builder.add_table_name(TableNames.TAG)
+            .add_select_conditions([f"{TableNames.TAG}.id"])
+            .add_sql_order_by_condition(["id"], self.query_builder.Order.ASC)
```

Comment:
it should be sorted by name, because when I ask it with offset it is returning me with a different order than when I don't send offset and that can cause problems in the pagination.